src/core/audio/AudioSdl.cpp

```diff
@@ -99,6 +99,9 @@ AudioSdl::AudioSdl( bool & _success_ful, Mixer*  _mixer ) :
 	_success_ful = true;
 
 #ifdef LMMS_HAVE_SDL2
+	// Workaround for a race condition that causes SDL to segfault
```

Comment:
Using a delay to fix a race condition sounds generally strange. But if it's the only option...

Can you give any source where you found the solution? Or is this just fixed out of your mind?

Reply:
The idea of adding a delay came out of my mind.

Audio devices are implemented as threads in SDL. Consequently, inserting a delay in theory gives the the playback device's thread time to "finish starting" before the capture device (and thread) is created, which prevents a race condition.

Reply:
Well, if it improves things, why not...